docs/command-line-tutorial.md

```diff
@@ -0,0 +1,324 @@
+```
+
+List the aggregators and identify the ID of both a leader and a helper.
+
+```sh
+divviup aggregator list
+```
+
+The output will contain JSON objects like:
+
+```json
+  {
+    "id": "3650870b-56e6-4eac-8944-b7ca36569b33",
+    "role": "Either",
+    "name": "Divvi Up staging-dap-09-1",
+    "dap_url": "https://staging-dap-09-1.api.example.com/",
+    "api_url": "https://staging-dap-09-1.api.example.com/aggregator-api",
+    "is_first_party": true,
+    "vdafs": [
+      "Prio3Count",
+      "Prio3Sum",
+      "Prio3Histogram",
+      "Prio3SumVec"
+    ],
+    "query_types": [
+      "TimeInterval",
+      "FixedSize"
+    ],
+    "protocol": "DAP-09",
+    "features": [
+      "TokenHash",
+      "TimeBucketedFixedSize",
+      "UploadMetrics"
+    ]
+  },
+  {
+    "id": "96301951-c848-4a57-b4f5-32812e4db1be",
+    "account_id": null,
+    "created_at": "2024-03-21T22:47:15.467139Z",
+    "updated_at": "2024-04-18T17:33:30.439465Z",
+    "deleted_at": null,
+    "role": "Either",
+    "name": "Divvi Up staging-dap-09-2",
+    "dap_url": "https://staging-dap-09-2.api.example.com/",
+    "api_url": "https://staging-dap-09-2.api.example.com/aggregator-api",
+    "is_first_party": false,
+    "vdafs": [
+      "Prio3Count",
+      "Prio3Sum",
+      "Prio3Histogram",
+      "Prio3SumVec"
+    ],
+    "query_types": [
+      "TimeInterval",
+      "FixedSize"
+    ],
+    "protocol": "DAP-09",
+    "features": [
+      "TokenHash",
+      "UploadMetrics",
+      "TimeBucketedFixedSize"
+    ]
+  }
+```
+
+Set the two IDs into environment variables. NOTE: These IDs will vary based on
+your configuration.
+
+```sh
+# Substitute the leader and helper aggregator IDs that were generated by your Divvi Up instance
+export LEADER_ID=3650870b-56e6-4eac-8944-b7ca36569b33
+export HELPER_ID=96301951-c848-4a57-b4f5-32812e4db1be
+```
+
+Next, generate a collector-credential for the task. The collector credential
+will be used by the collector to export the aggregated statistics.
+
+```sh
+divviup collector-credential generate --save
+```
+
+The output will be like:
+
+```sh
+{
+  "id": "0a0f8ea8-b603-4416-b138-b7f217153bb7",
+  "hpke_config": {
+    "id": 144,
+    "kem_id": "X25519HkdfSha256",
+    "kdf_id": "HkdfSha256",
+    "aead_id": "Aes128Gcm",
+    "public_key": "V9IpdJxS91MHPiNTjwDk9DFS-5M_neVrPxlmvolmTTo"
+  },
+  "created_at": "2024-05-03T15:23:56.624726Z",
+  "deleted_at": null,
+  "updated_at": "2024-05-03T15:23:56.624727Z",
+  "name": "collector-credential-144",
+  "token_hash": "VItYJdAyWYIvooe8GzkGnVTvaMkvWc9G-eiwxudfWww",
+  "token": "A6JDAYPiYDXmNyh-OpYXGw"
+}
+
+Saved new collector credential to /your/current/directory/collector-credential-144.json. Keep this file safe!
+```
+
+**Note**: This credential isn't sensitive because it's only useful in your local
+instance of Divvi Up. But collector credentials generated against the production
+service should be carefully managed. Consider using a password manager to
+protect them.
```

Comment:
Consider using the admonition syntax of Docusaurus https://docusaurus.io/docs/markdown-features/admonitions.

```suggestion
:::warning

This credential isn't sensitive because it's only useful in your local
instance of Divvi Up. But collector credentials generated against the production
service should be carefully managed. Consider using a password manager to
protect them.

:::
```

n.b. the extra spaces are important to prevent `prettier` from mangling this.